Source/relay/TourGuide/Sets/Fax.ash

```diff
@@ -22,11 +49,11 @@ string [int] SFaxGeneratePotentialFaxes(boolean suggest_less_powerful_faxes, boo
             //It's time for a g-g-g-ghost! zoinks!
             if (!__quest_state["Level 13"].state_boolean["digital key used"] && ($item[digital key].available_amount() + creatable_amount($item[digital key])) == 0)
```

Comment:
i removed this in the last main push because it isn't useful in any path except (i think?) exploathing at this point w/ the new digital key. good to remove this overall i think.

Reply:
Dropped it entirely then resolved the conflict.